### ISSUE TITLE
X34 support

### ISSUE THREAD

**Issue by RedKage** [thomas-crane/nrelay#55]
### Version info
 + nrelay cli version: `1.6.0`
 + nrelay version: `8.8.8`
 + RotMG version: `X34.0.0 1590058708`

### Plugins
My own, which connects to nexus and goes to quest area.

### Steps to reproduce
1. Start nrelay
2. Observe player connecting to nexus
3. Something is wrong

### Result
```
[12:16:24 | Updater]          Updating client and assets...
[12:16:24 | Updater]          Downloading assets...
[12:16:24 | Updater]          Fetching client version...
[12:16:25 | Updater]          Downloading client...
[12:16:25 | Updater]          Updated assets!
[12:16:30 | Updater]          Extracting packets...
[12:16:30 | Updater]          Extracting parameters...
[12:16:30 | Updater]          Updated packets and version!
[12:16:30 | Runtime]          Finished updating!
[12:16:30 | ResourceManager]  Loaded 1973 tiles.
[12:16:30 | ResourceManager]  Loaded 15184 objects.
[12:16:30 | Runtime]          Mapped 99 packet ids.
[12:16:30 | Runtime]          Using build version "X34.0.0"
[12:16:30 | Runtime]          Using client token "XTeP7hERdchV5jrBZEYNebAqDPU6tKU6"
[12:16:30 | LibraryManager]   Loading plugins...
[12:16:30 | LibraryManager]   Loading CollectorPlugin...
[12:16:30 | LibraryManager]   Loaded REDACTED_PLUGIN_NAME
[12:16:30 | Runtime]          Loading REDACTED...
[12:16:30 | AccountService]   Loading server list...
[12:16:30 | AccountService]   Cached server list loaded!
[12:16:30 | AccountService]   Loading character info...
[12:16:31 | AccountService]   Character info loaded!
[12:16:31 | Runtime]          Loaded REDACTED!
[12:16:31 | REDACTED]          Starting connection to EUWest
[12:16:31 | Runtime]          Loading REDACTED...
[12:16:31 | AccountService]   Loading server list...
[12:16:31 | AccountService]   Cached server list loaded!
[12:16:31 | AccountService]   Loading character info...
[12:16:31 | REDACTED]          Connected to EUWest!
[12:16:31 | REDACTED]          Received PacketIO error: The value of "offset" is out of range. It must be >= 0 and <= 101. Received 1039990887
```

**Comment by RedKage** [comment]
It looks like that to this day, this issue doesn't happen anymore.
Maybe this was fixed in an nrelay update.

**Comment by thomas-crane** [comment]
Yep, this was fixed in 8.8.9
